### ISSUE TITLE
Clarification regarding x_bar in incd_ablation_expt.py

### ISSUE THREAD

**Issue by shivanand-kundargi** [OatmealLiu/class-iNCD#8]
line.no 1119 in incd_ablation_expt.py 
       for batch_idx, ((x, x_bar), label, idx) in enumerate(tqdm(train_loader)):

       while performing experimentation on custom dataset, unable to unpack x_bar error throwsup, Please clarify by giving a brief 
       explanation regarding x_bar and its implications.
